src/stateMachine/InterfacingStateMachine.cpp

```diff
@@ -1,6 +1,5 @@
-#include "config/config.h"
+
```

Comment:
I don't think this should be removed

Reply:
Good point. I'll revert this.

Reply:
Actually, this one is actually not used. Do we still want to keep it?

Reply:
I think so, as we might forget to add it back if we use it in the future.